packages/account/src/Components/api-token/_tests_/api-token.spec.tsx

```diff
@@ -412,20 +338,20 @@ describe('<ApiToken/>', () => {
 
         render(
             <StoreProvider store={store}>
-                <ApiToken {...mock_props} />
+                <ApiToken />
             </StoreProvider>
         );
 
-        expect((await screen.findAllByText('Name')).length).toBe(3);
-        expect((await screen.findAllByText('Last Used')).length).toBe(3);
-        expect((await screen.findAllByText('Token')).length).toBe(3);
-        expect((await screen.findAllByText('Scopes')).length).toBe(3);
+        expect(await screen.findAllByText('Name')).toHaveLength(3);
+        expect(await screen.findAllByText('Last Used')).toHaveLength(3);
+        expect(await screen.findAllByText('Token')).toHaveLength(3);
+        expect(await screen.findAllByText('Scopes')).toHaveLength(3);
```

Comment:
```suggestion
        const api_token_titles = ['Name', 'Last Used', 'Token', 'Scopes'];
        api_token_titles.forEach(title => {
             expect(await screen.findAllByText(title)).toHaveLength(3);
        })
```

Reply:
just curious, does this make it faster or just cleaner?

Reply:
it makes the code to look more cleaner 😁